src/cmake/externalpackages.cmake

```diff
@@ -223,7 +223,7 @@ if (USE_CUDA OR USE_OPTIX)
         message (STATUS "CUDA_TOOLKIT_ROOT_DIR = ${CUDA_TOOLKIT_ROOT_DIR}")
     endif ()
 
-    find_package (CUDA 7.0 REQUIRED)
+    find_package (CUDA 8.0 REQUIRED)
```

Comment:
Curiosity: what is in this patch that required raising the Cuda floor? Do you think we'll end up sticking with 8.0? Or do you anticipate that it will likely creep up to 9?

Reply:
Nothing required it, but since I have only tested 8.0+ I decided to narrow the aperture somewhat.